tests/namespace_delete.go

```diff
@@ -112,11 +112,10 @@ func (s *namespaceTestSuite) SetupTest() {
 }
 
 func (s *namespaceTestSuite) Test_NamespaceDelete_InvalidUTF8() {
-	dc := s.cluster.host.dcClient
 	// don't fail for this test, we're testing this behavior specifically
-	dc.OverrideValue(s.T(), dynamicconfig.ValidateUTF8FailRPCRequest, false)
-	dc.OverrideValue(s.T(), dynamicconfig.ValidateUTF8FailRPCResponse, false)
-	dc.OverrideValue(s.T(), dynamicconfig.ValidateUTF8FailPersistence, false)
+	s.cluster.host.OverrideDCValue(s.T(), dynamicconfig.ValidateUTF8FailRPCRequest, false)
+	s.cluster.host.OverrideDCValue(s.T(), dynamicconfig.ValidateUTF8FailRPCResponse, false)
+	s.cluster.host.OverrideDCValue(s.T(), dynamicconfig.ValidateUTF8FailPersistence, false)
```

Comment:
should these use `s.overrideDynamicConfig` also?

Reply:
This test suite does not use FunctionalTestBase. Probably it can be refactored to use it.